### DISCUSSION TITLE
Update for 3095

### DISCUSSION THREAD

**Discussion by SlavicBlin** [tupoy-ya/TupoyeMenu#83] [Q&A]
I have updated Pointers, globals and classes for the latest version however when trying to join a gta online lobby it crashes while trying to load "Maintransition.hpp" i see that Yimmenu does not have this problem nor do they have "Maintransition.hpp" so iam wondering what would happend if i removed "Maintransition.hpp" since from what i can tell its used for Seamless joining and Seamless joining is bit buggy anyways.

**Comment by tupoy-ya** [comment]
Yes, you can remove it and remove everything related to it in the `src/native_hooks/native_hooks.cpp` file.
( I think it has RID joining in it as well)

I will probably release an updated version in the next few weeks, when YimMenu is stable and i have time for this game.

**Reply by SlavicBlin** [reply]
cool cheers for the quick response 👍

**Reply by SlavicBlin** [reply]
Yeah i removed it and now it works just fine 👍
